platforms/hyperledger-indy/charts/indy-ledger-txn/templates/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
not needed, the configmap is already created by https://github.com/hyperledger-labs/blockchain-automation-framework/blob/0af031b0db2f9c945deca42f0a9a92c333df522e/platforms/hyperledger-indy/configuration/deploy-network.yaml#L131